sql-plugin/src/main/scala/com/nvidia/spark/rapids/shuffle/RapidsShuffleClient.scala

```diff
@@ -355,6 +385,33 @@ class RapidsShuffleClient(
     clientCopyExecutor: Executor,
     maximumMetadataSize: Long) extends Logging {
 
+  private[this] var devStorage: RapidsDeviceMemoryStore = GpuShuffleEnv.getDeviceStorage
+  private[this] var catalog: ShuffleReceivedBufferCatalog = GpuShuffleEnv.getReceivedCatalog
```

Comment:
Nit: I'd prefer to make these `val` and have the standard constructor default to these values rather than mutate them with an alternate constructor.

Reply:
Nice, that's a lot better.